path_config.go

```diff
@@ -33,6 +34,10 @@ func pathConfig(b *jwtAuthBackend) *framework.Path {
 				Type:        framework.TypeCommaStringSlice,
 				Description: `A list of PEM-encoded public keys to use to authenticate signatures locally. Cannot be used with "oidc_discovery_url".`,
 			},
+			"jwt_supported_algs": {
+				Type:        framework.TypeCommaStringSlice,
+				Description: `A list of supported encoding algorithms. Defaults to empty, therefore the oidc plugin default, which currently is RS256 only.`,
```

Comment:
```suggestion
				Description: `A list of supported signing algorithms. Defaults to RS256.`,
```

Reply:
@kalafut LGTM, let me know if you'd like me to rebase and commit this text change, or if you prefer to just squash it on merge.

Reply:
You should have been given a button to just accept the change in the GH UI which will automagically make a new commit. Don't rebase or squash. We'll do that at the end.